config/repo-values.yaml

```diff
@@ -99,5 +99,7 @@ sync:
       url: https://charts.appuio.ch
     - name: gomods
       url: https://athens.blob.core.windows.net/charts
+    - name: billimek
+      url: https://billimek.github.io/helm-repo
```

Comment:
The `url` field in both files need to be the same. We plan on eventually generating one file from the other so it's important that they match.

Reply:
I wasn't sure if both URL values should be the repo with the index, or one should be a more human-readable version.  This is now fixed!